libdnf/utils/CompressedFile.cpp

```diff
@@ -15,7 +18,14 @@ CompressedFile::~CompressedFile() = default;
 
 void CompressedFile::open(const char *mode)
 {
+    errno = 0;
```

Comment:
Please could you add description what you do here? Believe that this is related to #include <cerrno>.

Reply:
There are situations when `solv_xfopen` return `NULL` but it does not set `errno` -> `errno` value is not defined in this case. So, `errno` is initialized to zero before `solv_xfopen` is called.
I will add comment to code.